pkg/util/docker/containers.go

```diff
@@ -190,10 +191,28 @@ func parseContainerHealth(status string) string {
 
 // parseContainerNetworkAddresses converts docker ports
 // and network settings into a list of NetworkAddress
-func parseContainerNetworkAddresses(ports []types.Port, netSettings *types.SummaryNetworkSettings, container string) []containers.NetworkAddress {
+func (d *DockerUtil) parseContainerNetworkAddresses(cID string, ports []types.Port, netSettings *types.SummaryNetworkSettings, container string) []containers.NetworkAddress {
```

Comment:
I have the bridge mode/host mode handling happening further up in DockerUtil.List() in https://github.com/DataDog/datadog-agent/pull/4418  -- should that move here as well?

Reply:
yep, can do 👍